pkg/storages/azure/folder.go

```diff
@@ -91,10 +90,10 @@ func ConfigureFolder(prefix string, settings map[string]string) (storage.Folder,
 		environmentName = defaultEnvName
 	}
 
-	var credential azcore.Credential
+	var credential *azblob.SharedKeyCredential
 	var err error
 	if usingToken {
-		credential = azcore.NewAnonymousCredential()
+		credential = nil
```

Comment:
`credential` is already `nil`, isn't it?

Reply:
Yes. I'm used to languages requiring assignment before use